src/activities/assignments/AssignmentEntity.js

```diff
@@ -610,6 +614,22 @@ export class AssignmentEntity extends Entity {
 			fields.push({ name: 'completionType', value: assignment.completionType });
 		}
 
+		if (typeof assignment.isIndividualAssignmentType !== 'undefined') {
+			if (assignment.isIndividualAssignmentType !== this.isIndividualAssignmentType()) {
+				fields.push({ name: 'isIndividualAssignmentType', value: assignment.isIndividualAssignmentType });
+			}
+
+			if (!assignment.isIndividualAssignmentType && !this.isGroupAssignmentTypeDisabled()) {
+				fields.push({ name: 'groupTypeId', value: assignment.groupTypeId });
+				fields.push({ name: 'folderType', value: 1 });
+			} else {
+				const action = this._getActionSetToIndividualAssignmentType();
```

Comment:
I _super_ do not like this. I would prefer you just hard-code `folderType = 2` and `groupTypeId = null`